guides/lagom-dcos-marathon-deploy-microservices/src/main/paradox/index.md

```diff
@@ -0,0 +1,392 @@
+# Deploying Lagom Microservices on DC/OS
+
+<style type="text/css">
+  pre.code-bash::before {
+    content: '$ ';
+    color: #009900;
+    font-weight: bold;
+  }
+</style>
+
+[Lagom](http://www.lagomframework.com/) is a flexible microservices framework that makes it quick and easy to
+build, test, and deploy your systems with confidence. [DC/OS](https://dcos.io/), an open-source distributed operating
+system based on [Apache Mesos](http://mesos.apache.org/), provides features that complement running Lagom applications
+in production. This guide will cover the configuration required to run your Lagom-based system on DC/OS using its
+built-in [Marathon](https://mesosphere.github.io/marathon/) and [Docker](https://www.docker.com/) support.
+
```

Comment:
This guide uses the Lagom Chirper example , which is already configured to run on DC/OS. It points out the configuration you would need to make in your Lagom system. And, it provides step-by-step instructions for running the Chirper example on DC/OS.